### DISCUSSION TITLE
Electrical heater state/power ? Water Circulation pump state?

### DISCUSSION THREAD

**Discussion by Kifo2023** [john30/ebusd#994] [Q&A]
I am using an Vaillant AroTherm plus VWL 105/6A (config HMU00),
and a Vaillant VRC 720 sensoComfort (config CTLV2).

So far I could not find any data regarding the electircal supplementary heating rod ("Heizstab").

Any idea which parameter might reveal information about it?
Same for my hot water circulation pump. Seems it is connected like this: "HMU MultiRelaySetting: Circulation". But I cannot find any on/off information.

Thanks a lot.

**Comment by rICTx-T1D** [comment]
Hi

> I am using an Vaillant AroTherm plus VWL 105/6A (config HMU00), and a Vaillant VRC 720 sensoComfort (config CTLV2).

I have simular system.
> 
> So far I could not find any data regarding the electircal supplementary heating rod ("Heizstab").

Details from "ImmersionHeater" (German Heistab) you find in vwzio-Device.

![image](https://github.com/user-attachments/assets/7cc5d072-f34a-4efc-b719-2206c496b269)


> 
> Any idea which parameter might reveal information about it? Same for my hot water circulation pump. Seems it is connected like this: "HMU MultiRelaySetting: Circulation". But I cannot find any on/off information.


This is only a relay they can be conifure in GUI of device.

> Thanks a lot.
Hope the answer is not to late :).

**Reply by Kifo2023** [reply]
Which configuration (csv) file are you using to accezz the vwzio device?
